purchases/src/main/kotlin/com/revenuecat/purchases/galaxy/GalaxyBillingWrapper.kt

```diff
@@ -81,9 +110,7 @@ internal class GalaxyBillingWrapper(
         TODO("Not yet implemented")
     }
 
-    override fun isConnected(): Boolean {
-        TODO("Not yet implemented")
-    }
+    override fun isConnected(): Boolean = true
```

Comment:
As far as I can tell, connecting/disconnecting isn't really a thing for the Samsung SDK so long as you have an instance of the `IapHelper` class